app/Database/CourseQueries.hs

```diff
@@ -45,17 +45,22 @@ retrieveCourse = liftIO . queryCourse . T.pack
 
 -- | Queries the database for all information about @course@, constructs a JSON object
 -- representing the course and returns the appropriate JSON response.
-queryCourse :: T.Text -> IO Response
+queryCourse :: T.Text -> IO Response -- change T.Text to courses Key
 queryCourse str = do
     courseJSON <- returnCourse str
     return $ createJSONResponse courseJSON
 
 -- | Queries the database for all information about @course@,
 -- constructs and returns a Course value.
-returnCourse :: T.Text -> IO Course
+returnCourse :: T.Text -> IO Course -- change t.text to courses key
 returnCourse lowerStr = runSqlite databasePath $ do
+<<<<<<< HEAD
```

Comment:
@klmerryn this text is a sign of a merge conflict that you needed to resolve before committing. Basically there are two versions of the same code, and you need to merge them into one. Make sure to keep the changes pull from master (I'm not sure you actually changed anything important here).